scripts/inject-protocol.sh

```diff
@@ -32,6 +32,7 @@ function fetch_protocol {
 ---
 # This file is generated by _scripts/inject-protocol.sh. Do not modify it manually!
 title: Resumable upload protocol ${slug}
+description: "The Tus protocol specification: Detailing the interactions between Tus clients and servers for reliable and resumable file uploads."
```

Comment:
We do use the capitalized `Tus` on transloadit.com, but on tus.io we prefer `tus` for now :)